CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## [2.30.2024022914] (2024-02-29)
```

Comment:
@dbanck Where is this version number come from?

Reply:
I figured, but you did commit that number to the changelog, right? So you ran that locally and then pasted the version in there? Or did you anticipate the number based on the time you made the commit?
(I'm poking for details here, as that is lacking in the guide on creating the changelog 😄)

Reply:
🪄

Reply:
The process of maintaining a changelog for pre-releases needs some improvement. I created the changelog commit, ran the script (which uses the commit date), copied the version, and amended the commit.